src/server/journal/journal_streamer.h

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
```

Comment:
minor not, the files inside journal are not prefixed with journal_

Reply:
mmm I can change this though there is - journal slice